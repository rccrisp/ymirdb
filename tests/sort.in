set a 15 -7 3 2
sort a
get a
bye
