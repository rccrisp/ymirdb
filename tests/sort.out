> ok

> ok

> [-7 2 3 15]

> bye
